client/write.go

```diff
@@ -98,11 +100,12 @@ type writeState struct {
 	sync.Mutex
 	refCounter
 
-	session             *session
-	op                  *writeOp
-	majority            int32
-	successful, pending int32
-	errors              []error
+	session           *session
+	topoMap           topology.Map
+	op                *writeOp
+	majority, pending int32
+	success           int32
```

Comment:
They interact with a bunch of int32's in session.go, so this avoids casting all over the place.